test/ExecutionFramework.h

```diff
@@ -51,6 +51,25 @@ class ExecutionFramework
 {
 
 public:
+	struct log_record
+	{
+		size_t index;
+		/// The address of the account which created the log.
```

Comment:
Most of these comments are redundant. What I would like to know the meaning of is `index`. Is it its position in the transaction or the call?